mcs/class/corlib/System.Reflection/FieldInfo.cs

```diff
@@ -263,6 +263,9 @@ internal CustomAttributeData[] GetPseudoCustomAttributesData ()
 			return attrsData;
 		}
 
+		[MethodImplAttribute(MethodImplOptions.InternalCall)]
+		internal static extern int get_metadata_token (FieldInfo fieldInfo);
```

Comment:
This will require mscorlib version bump, see https://github.com/mono/mono/blob/master/configure.ac#L43